src/Compiler/Checking/CheckPatterns.fs

```diff
@@ -678,7 +678,13 @@ and TcPatLongIdentUnionCaseOrExnCase warnOnUpper cenv env ad vFlags patEnv ty (m
         elif numArgs < numArgTys then
             if numArgTys > 1 then
                 // Expects tuple without enough args
-                errorR (Error (FSComp.SR.tcUnionCaseExpectsTupledArguments numArgTys, m))
+                let missingArgs = 
+                    argNames.[numArgs..numArgTys - 1]
+                    |> List.map (fun id -> (if id.rfield_name_generated then "" else id.DisplayName + ": ") + id.FormalType.ToString())
+                    |> String.concat "\n, "
```

Comment:
Should line endings be platform specific? I.e. `Environment.NewLine`?

Reply:
In Unix platforms, it is \n anyway.
And in VS on Windows, it works in the rendered popup:

![image](https://user-images.githubusercontent.com/46543583/192272002-466bb605-3bc8-4c0c-b334-231b5087cf90.png)


But I will modify, sure.

Reply:
Also, it's an unusual comma placement. Are we sure we want it there? :)

Reply:
Thanks @nojaf , good catch. Indeed, using NicePrint.minimalStringOfType env.DisplayEnv will be more systematic here.

Reply:
![image](https://user-images.githubusercontent.com/46543583/192278766-36f65618-12a7-4c3a-8cf9-8cbb8b85ac3d.png)